uwblueprint/project-read-backend
# Add enrolments field to FamilyDetailSerializer

### Notion ticket link



Previous enrolment section (https://www.notion.so/uwblueprintexecs/Developer-Hub-30e9b6c82af24054b31acf7c5e822c89?p=54f32176f7624da7acfec32dc7f9bfbf)



### Implementation description

- added enrolments as a serializer method field on the family model 
- wrote get_enrolments function to call the enrolment serializer and return a list of enrolments that a particular family has been enrolled in 



### Steps to test

1. pull 
2. in the API docs, run the families/{id} GET endpoint
3. verify that a family has a populated enrolments field



### What should reviewers focus on?

- should this be filtered to include only statuses that indicate the class was in the past? (e.g. if an enrollment's status is registered / signed up / current is this relevant for GSL to know (this is a past interaction, but not a past class attended)). 

### Checklist
